### ISSUE TITLE
FR37 desktop GIS access to LBIMS data

### ISSUE THREAD

**Issue by gubuntu** [kartoza/LEDET_BIMS#40]
# Problem

payment of the next milestone invoice depends on LEDET staff being able to consume OWS services and connect to PostGIS on production from QGIS and ArcGIS

# Solution

Test and confirm __TODAY and up to 25 Sept__ that this is the case. Connect via VPN, ask LEDET staff to test on your behalf, etc. 

- [x] GIS layers via OGC services
- [x] GIS layers via direct database connection

## Prerequisites
- users and groups are set up in PostGIS (#41)
- port 5432 on the geonode db is exposed and pg_hba.conf and postgresql.conf are properly configured . pg_hba should only allow connections from within the WAN / VPN and also force  SSL.

depends on #168
depends on #41

**Comment by meomancer** [comment]
Bims layer is using list because of backbone formatter on javascript level.
We need a parameter that tells format for this
Need to update the API

Maybe in another ticket?

CC @gubuntu @NyakudyaA

**Comment by NyakudyaA** [comment]
Vince can now connect to the Database although they are some network issues on his side.
![screenshot 2018-10-23 at 08 55 26](https://user-images.githubusercontent.com/2510900/47341525-84269580-d6a1-11e8-9ca6-13e589575c86.png)

OWS Services can also be accessed by Vince.
![screenshot 2018-10-23 at 08 58 59](https://user-images.githubusercontent.com/2510900/47341668-e8e1f000-d6a1-11e8-8e40-292f5b434b39.png)
